src/services/login.service.ts

```diff
@@ -0,0 +1,24 @@
+import jwt from 'jsonwebtoken';
+import UserModel from '../models/user.model';
+
+const SECRET_KEY = 'your_secret_key'; 
```

Comment:
why SECRET_KEY defined here? move it in constant file

Reply:
Resolved